cgrok/grok.h

```diff
@@ -4,8 +4,10 @@
 #ifndef _GROK_H_
 #define _GROK_H_
 
+#include <stdio.h>
+#include "tree.h"
```

Comment:
! [Nice one](http://static4.fjcdn.com/thumbnails/comments/See+you+on+the+frontpage+_9ba0ce131339c97e036990761e0f8456.gif)